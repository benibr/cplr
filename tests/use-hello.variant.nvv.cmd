../bin/cplr 'puts("Hello.")' -nvv
