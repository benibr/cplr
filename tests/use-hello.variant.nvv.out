Preparation phase
Generation phase
Generating code
Generating section defsysinclude
Generating section statements
Generated: 1048 bytes code, 0 bytes dump
Compilation phase
Compilation succeeded
